core/src/main/java/org/infinispan/conflict/impl/ConflictMangerModuleLifecycle.java

```diff
@@ -0,0 +1,27 @@
+package org.infinispan.conflict.impl;
+
+import org.infinispan.configuration.cache.Configuration;
+import org.infinispan.conflict.ConflictManager;
+import org.infinispan.factories.ComponentRegistry;
+import org.infinispan.lifecycle.AbstractModuleLifecycle;
+import org.infinispan.lifecycle.ModuleLifecycle;
+import org.kohsuke.MetaInfServices;
+
+/**
+ * @author Ryan Emerson
+ * @since 9.1
+ */
+@SuppressWarnings("unused")
+@MetaInfServices(value = ModuleLifecycle.class)
+public class ConflictMangerModuleLifecycle extends AbstractModuleLifecycle {
```

Comment:
Actually, I've changed approach and removed this file. Now the `ConflictManager` and `StateReceiver` are now created via the `StateTransferComponentFactory`.